dave/template.md

```diff
@@ -0,0 +1,68 @@
+# My Corp Inc.
```

Comment:
should dave be in `examples/dave` ? we already have john, why do we need dave? I think its to demo the revision command but this should be documented in the PR description